### ISSUE TITLE
Provide automatic fine tuning 

### ISSUE THREAD

**Issue by 2013-11390** [omnis-labs-company/omnis#335]
https://arxiv.org/abs/1811.08883v1
We should give proper hyper parameter during training. 
For example, we can give high learning rate in the early training and give small learning rate in the later training

**Comment by 2013-11390** [comment]
https://machinelearningmastery.com/using-learning-rate-schedules-deep-learning-models-python-keras/
